staging/kos/README.md

```diff
@@ -139,6 +142,25 @@ The SDN has just four concepts, as follows.
 A controller chooses the IP address for each NetworkAttachment.  An
 attachment's MAC address is a function of the IP address and the VNI.
 
+Neither controller enforces any interlock between its actions and the
+lifecycle of any API object. Consequently, there can not be any fully
+effective enforcement of immutability of any field of any API object ---
+the client can always delete and object and create a replacement with the
+same name, possibly without the controller seeing any intermediate state.
+Coping with all possible changes makes for complicated controller code.
+Stay tuned for lifecycle interlocks.
+
+Neither controller enforces or assumes any connections between the
+lifecycles of NetworkAttachments and their Subnets; either can be
+created or deleted at any time.  There are, however, some intended
+consistency constraints between certain fields of certain objects.
+The apiservers attempt to enforce this consistency, but this
+enforcement is necessarily imperfect because there are no ACID
+transactions that involve more than one object.  The controllers
+therefore must be prepared to react safely if and when an
+inconsistency gets through.  Again, this makes for very complicated
+controller code and this area is still work in progress.
```

Comment:
I think "still a work in progress" is a weaker statement than we now intend.

Reply:
Removed "still work in progress" in otherwise unrelated PR #69.